src/dnddiceroller/DnDDiceRoller.java

```diff
@@ -0,0 +1,80 @@
+package dnddiceroller;
+
+import assignment.pkg9.DiceFileReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+import java.util.Scanner;
+import java.util.StringTokenizer;
+
+/**
+ *
+ * @author Paul Scarrone
+ */
+public class DnDDiceRoller {
+
+  /**
+   * Try out your dies and dice tower
+   * @param args the command line arguments
+   */
+  public static void main(String[] args) throws FileNotFoundException {
+      for(int i = 0; i< 1000; i++){
+	  int d6 = test_oneD6();
+	  if(d6 != -1){
+		System.out.println("Die Test Failed with Value: " + d6);
+	  }
+	  int tower = test_diceTowerWithTwoD6();
+	  if(tower != -1){
+	System.out.println("Tower Test Failed with Value: " + tower);
+	  }
+          DiceFileReader diceReader = diceReader = new DiceFileReader("./dice.txt");
+	}
+      Scanner scanner = null;
+      ArrayList<String> list = new ArrayList<String>();
+        File temp = new File ("dice.txt");
+        Scanner file = new Scanner (temp);
+        String s = scanner.nextLine();
+   while(file.hasNext()){
+       String[] tempList=s.split(":");
```

Comment:
Before you deal with tokenizing the file you will need to exclude the lines of the file that do not have useful information starting with `####`